airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -73,6 +73,13 @@ def _get_init_containers(self):
             'name': 'GIT_SYNC_REV',
             'value': self.kube_config.git_sync_rev
         }]
+
+        for env_var_name, env_var_val in six.iteritems(self.kube_config.kube_env_vars):
```

Comment:
```suggestion
        for env_var_name, env_var_val in self.kube_config.kube_env_vars.items():
```

Airflow dropped Python 2 support, so no need to use the six library. Other than that, looks good to me.

Reply:
This is targeted for the v1-10 branch